### ISSUE TITLE
[FEATURE] CTF | make team section - up to 5 players

### ISSUE THREAD

**Issue by ArkadiK94** [thecyberworld/TheCyberHUB#730]
### Detailed description

from setting / menu of registration or somewhere else, we can add a page for 
 - creating a team,
 - add users in a team, up to 5
 - can do chat with a team
 

### Context

we are also planing CTF in team, that will be helpful in that way

### Possible implementation

we can remove role, but except that it's good

![image](https://github.com/thecyberworld/TheCyberHUB/assets/76536506/6b4e1fcd-fc05-4592-b7a8-e05526c853b6)
https://dribbble.com/shots/23135912-Managing-and-creating-teams-web-roles

### Additional information

If you have any question, you are welcome to ask :)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by AhmedKarray005** [comment]
@ArkadiK94 you want to add a page where the user can make a team and add members to this team?

**Comment by ArkadiK94** [comment]
@AhmedKarray005 Yes, and/or ask to join an existing team.

**Comment by AhmedKarray005** [comment]
can u assign it to me @ArkadiK94 ?

**Comment by ArkadiK94** [comment]
@AhmedKarray005 If you have any question you are welcome to ask :)

**Comment by ArkadiK94** [comment]
@AhmedKarray005 Hi, how is it going?

**Comment by AhmedKarray005** [comment]
i am still working on it @ArkadiK94

**Comment by AhmedKarray005** [comment]
@ArkadiK94 u just want the front right?

**Comment by ArkadiK94** [comment]
@AhmedKarray005 yes, you can create dummy json file with dummy data if you need it.

**Comment by AhmedKarray005** [comment]
@ArkadiK94 is there servers problem , bcs i cannot connect to my account

**Comment by ArkadiK94** [comment]
@AhmedKarray005 which error do you get? Do you tried with username?

**Comment by AhmedKarray005** [comment]
@ArkadiK94 

https://github.com/thecyberworld/TheCyberHUB/assets/104280658/be188a44-d112-4ca6-9e80-8c92a783fbbf

**Comment by AhmedKarray005** [comment]
what to modify?

**Comment by AhmedKarray005** [comment]
@ArkadiK94 i did it

**Comment by AhmedKarray005** [comment]
@ArkadiK94 , you will find it on the display page of the settings section

**Comment by ArkadiK94** [comment]
#783 Some progress was done thanks.
@kabir0x23 Do you want me assign you in this issue about changing some styles around?
